packages/components/src/select-control/test/index.js

```diff
@@ -260,33 +260,27 @@ describe( 'SelectControl', () => {
 			);
 		} );
 
-		describe( 'prop excludeSelectedOptions', () => {
-			it( 'should preserve selected option when focused', async () => {
-				const onChangeMock = jest.fn();
-				const { getByRole } = render(
-					<SelectControl
-						isSearchable
-						showAllOnFocus
-						selected={ options[ 2 ].key }
-						options={ options }
-						onSearch={ () => options }
-						onFilter={ () => options }
-						onChange={ onChangeMock }
-						excludeSelectedOptions={ false }
-					/>
-				);
-				getByRole( 'combobox' ).focus();
-				await waitFor( () =>
-					expect(
-						getByRole( 'option', { name: 'bar' } )
-					).toBeInTheDocument()
-				);
-				// In browser, the <Button> in <List> component is automatically "selected" when <Control> lost focus.
-				// I was not able to produce the same behaviour with unit test, but a click on the currently
-				// selected option should be sufficient to simulate the logic in this test.
-				userEvent.click( getByRole( 'option', { selected: true } ) );
-				expect( onChangeMock ).toHaveBeenCalledTimes( 0 );
-			} );
```

Comment:
This test case should still pass, but broke because of the changes to the initial state above.

Reply:
True, this is now fixed in 0890a7b